chef/cookbooks/dhcp/templates/default/dhcpd.conf.erb

```diff
@@ -21,6 +21,34 @@ log-facility local7;
 # Fix for https://bugzilla.opensuse.org/show_bug.cgi?id=961536
 always-reply-rfc1048 true;
 
+<% if @ip_version == "6"  -%>
+# Other options we may want?
+#option dhcp6.rfc4833-tz-posix-string code 41 = string;
+#option dhcp6.rfc4833-tz-name code 42 = string;
+#
+# Use example:
+#   option dhcp6.rfc4833-tz-posix-string "EST5EDT4,M3.2.0/02:00,M11.1.0/02:00";
+#   option dhcp6.rfc4833-tz-name         "Europe/Zurich";
+# Use this to send dhcp log messages to a different log file (you also
+# have to hack syslog.conf to complete the redirection).
+
+# Set preference to 255 (maximum) in order to avoid waiting for
+# additional servers when there is only one
+##option dhcp6.preference 255;
+
+# Server side command to enable rapid-commit (2 packet exchange)
+##option dhcp6.rapid-commit;
+
+# The delay before information-request refresh
+#  (minimum is 10 minutes, maximum one day, default is to not refresh)
+#  (set to 6 hours)
+#option dhcp6.info-refresh-time 21600;
+
+include "/etc/dhcp3/groups.d/group6_list.conf";
+include "/etc/dhcp3/subnets.d/subnet6_list.conf";
+include "/etc/dhcp3/hosts.d/host6_list.conf";
+<% else -%>
```

Comment:
They are the default values from the "sample" config, there commented out there too. Really there there because maybe there something we may or may not what to tweak and left them here seeing as I haven't really testing the actual running dhcpd6 yet.. other then it starts.

Also it was WIP so figured doesn't hurt. I'll remove them for now.